scikits/fitting/__init__.py

```diff
@@ -77,14 +77,17 @@
 from spline import *
 
 
+logger = logging.getLogger(__name__)
+
+
 def _setup_test():
     """Create test() method that will run unit tests via nose."""
     args = ['', '--exe', '-w', __path__[0]]
     try:
         import nose as _nose
     except ImportError:
         def test():
-            print('Could not load nose. Unit tests not available.')
+            logger.debug('Could not load nose. Unit tests not available.')
```

Comment:
Will this message ever be visible i.e. is logging actually configured at this point?